xline/src/storage/kv_store.rs

```diff
@@ -937,6 +944,37 @@ mod test {
         Ok(())
     }
 
+    #[tokio::test(flavor = "multi_thread", worker_threads = 10)]
+    async fn test_kv_store_atomic() {
```

Comment:
Atomic usually means we can't insert other operations between the continuous operations in the logic view. I don't think the test tells the store is "atomic".

Reply:
Renamed